astropy/io/misc/asdf/tags/unit/unit.py

```diff
@@ -1,8 +1,6 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 # -*- coding: utf-8 -*-
 
-import six
```

Comment:
That is so weird! How did we miss this in our PY2 clean-up effort? And how is it not crashing since we don't require `six` as dependency anymore?

Reply:
This line is hit in the unit tests. The best explanation that I can think of is that matplotlib has a dependency on six via python-dateutil, so we may not be truly testing that this does not work without six.

Reply:
Thanks for the mention, we've already removed six from asdf-astropy so we're all caught up over there.